lib/messenger.rb

```diff
@@ -0,0 +1,30 @@
+module Messenger
+
+  def send_sms(number, content)
+    sid = ENV["TWILIO_ACCT_SID"]
+    auth = ENV["TWILIO_AUTH"]
+
+    @client = Twilio::REST::Client.new sid, auth
+
+    from = "+14123854063"
+
+    # The following try-rescue block is needed in case user unsubscribe
+    # if the user ubsubscribe and we attempt to message them
+    # the api will report an error
+
+    begin
+      message = @client.account.messages.create(
+          :from => from,
+          :to => '+1'+number,
+          :body => content
+      )
+    rescue Twilio::REST::RequestError => e
+      case e.code
+        when 21610
```

Comment:
I totally agree we should have the `try/rescue` I just think we should do more in the rescue beyond just logging the error, especially if we know what the error means and how to prevent it in the future (which isn't the case for every type of error, but is for this one)